### PR TITLE
Support string dtype for OV backend

### PR DESCRIPTION
**Pull request by andrey-churkin**
### Changes

- Support string dtype for OV backend

### Reason for changes

NotImplementedError: NNCF is not yet supported OpenVINO data type: string.

### Related tickets

136751

### Tests

-  test_convert_to_nncf_dtype_supported_types()
- test_convert_to_nncf_dtype_unsupported_types()

### CODE REVIEW

File: nncf/common/graph/layer_attributes.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 class Dtype(Enum):
     FLOAT = "float"
     INTEGER = "int"
+    STRING = "string"
```
Comment (rkazants) [inline nncf/common/graph/layer_attributes.py:22]: may you have test for it?

**Comment by DaniAffCH** [comment]
Hi @andrey-churkin could you please add the inverse mapping (from NNCF dtype to OV dtype) in the function `convert_nncf_dtype_to_ov_dtype` within `nncf/experimental/common/graph/netron.py`.
Otherwise, the Netron export won't work for networks containing `string` as `dtype`

File: nncf/common/graph/layer_attributes.py (hunk 1 above)
Comment (andrey-churkin) [reply nncf/common/graph/layer_attributes.py]: I have added tests to be sure that we cover all available OpenVINO data types.

**Comment by andrey-churkin** [comment]
@DaniAffCH Thanks! I have updated the code recently. So everything is fine now.

**Review by alexsu52 — APPROVED** [review]
LGTM

**Pull request merged**